jedi/evaluate/sys_path.py

```diff
@@ -33,8 +33,15 @@ def _get_venv_sitepackages(venv):
     if os.name == 'nt':
         p = os.path.join(venv, 'lib', 'site-packages')
     else:
-        p = os.path.join(venv, 'lib', 'python%d.%d' % sys.version_info[:2],
-                         'site-packages')
+        # Account for the VIRTUAL_ENV using an alternate python verison
+        venv_lib = os.path.join(venv, 'lib')
+        try:
+            pyver = [p for p in os.listdir(venv_lib) if
```

Comment:
Theoretically yes. I believe however that having multiple pythonX.X files in the lib directory would be a non-standard setup (certainly when calling virtualenv with or without '-p' you should get a single-entry list returned.

If you think this is a valid corner case to be accounted for then check_virtual_env() could be modified to deal with multiple results from _get_venv_sitepackages().